README.md

```diff
@@ -510,6 +510,7 @@ _Related: [Metrics & Metric Collection](#metrics--metric-collection)_
 - [NetXMS](https://www.netxms.org/) - Open Source network and infrastructure monitoring and management. ([Source Code](https://github.com/netxms/netxms)) `LGPL-3.0/GPL-3.0` `Java/C++/C`
 - [Observium](http://www.observium.org/) - SNMP monitoring for servers and networking devices. Runs on linux.
 - [OMD](http://omdistro.org/) - The Open Monitoring Distribution.
+- [openITCOCKPIT](https://openitcockpit.io/) - Monitoring Suite featuring seamless integrations with Naemon, Checkmk, Grafana and more. ([Demo](https://demo.openitcockpit.io/), [Source Code](https://github.com/it-novum/openITCOCKPIT)) `GPL-3.0/Proprietary` `PHP`
```

Comment:
```suggestion
- [openITCOCKPIT](https://openitcockpit.io/) - Monitoring Suite featuring seamless integrations with Naemon, Checkmk, Grafana and more. ([Demo](https://demo.openitcockpit.io/), [Source Code](https://github.com/it-novum/openITCOCKPIT)) `GPL-3.0` `deb/Docker`
```

Reply:
> Based on your Documentation, it appears that you do not support manual installation of the software. Instead, you require it to be installed either through packages or Docker. Consequently, I have adjusted the language tags accordingly.


That's correct. openITCOCKPIT bundels a lot of open source tools such as [Naemon](https://github.com/naemon/naemon-core/), [Statusengine](https://github.com/statusengine/),
[Graphite](https://github.com/graphite-project) or [Grafana](https://github.com/grafana/grafana) just to name a few.
Some of those needs to be compiled other just configured. It's a complex setup which is why we do not provide any manual installation documentation.

On the other hand, provide some [documentation about how to setup a development system](https://docs.openitcockpit.io/en/development/setup-dev-env/), be clonen the source code from GitHub.

> Given that the default installation seems to be the CE version, it seems appropriate to omit references to "proprietary" licenses. 

That's fine for me, by default a user will always end up with Community Edition :)
My idea was to mention the dual license because it is also mentioned in the projects readme.